pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -329,8 +328,16 @@ def read_mwa_corr_fits(
                 raise ValueError("only fits, metafits, and mwaf files supported")
 
         # checks:
-        if metafits_file is None:
+        if metafits_file is None and ppds_file is None:
             raise ValueError("no metafits file submitted")
+        elif metafits_file is None:
+            metafits_file = ppds_file
+        elif ppds_file is not None:
+            ppds = fits.getheader(ppds_file, 0)
+            meta = fits.getheader(metafits_file, 0)
+            for key in ppds.keys():
+                if key not in meta.keys():
+                    self.extra_keywords[key] = ppds[key]
```

Comment:
What happens if just a metafits file is given (with no ppd file)? I don't see a section where that's handled and I can't tell whether or not any of the existing tests test that.

Reply:
If just the metafits file is given, it has the old default behavior which expects a metafits file.

Reply:
Probably the proper thing for me to do is implement the test I procrastinated; doing a full cotter compare instead of just the data reordering and the cable length correction.

Reply:
I think it's ok if you have a test already that only takes in the metafits file and not the ppd file, I just couldn't tell if that was the case.

Reply:
Yes, `test_read_mwa_write_uvfits` does this.